docs/source/index.rst

```diff
@@ -134,6 +134,24 @@ torchtune tutorials.
    deep_dives/recipe_deepdive
    deep_dives/wandb_logging
 
+.. toctree::
+   :glob:
+   :maxdepth: 1
+   :caption: General Usage
```

Comment:
Thinking there could be a better caption than "General Usage". Some otions:
* Concepts and Operations
* Torchtune Basics
* User Guide
* HowTos 

Also I think it should probably go after Finetuning recipes?

Reply:
Yes I like Torchtune Basics or just Basics. and that location sounds better